### PR TITLE
Fix runtime warning on pytorch 1.6

### PR DESCRIPTION
**Pull request by BloodAxe**
## Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)?
- [x] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [x] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [x] Did you check the docs with `make check-docs`?
- [x] Did you write any new necessary tests?
- [x] Did you add your new functionality to the docs?
- [x] Did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/CHANGELOG.md)?
- [x] **You can use 'Login as guest' to see Teamcity build logs.**

## Description

Fixes runtime warning in PyTorch 1.6 in distributed training mode caused by creation of tensor copy from another tensor.
```
# To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or
# sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
```

## Related Issue

None

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by Scitator** [comment]
@BloodAxe could you please re-check the codestyle?

**Comment by BloodAxe** [comment]
Btw, @Scitator it seems code-style tooling is unix-only and there is no window-friendly support for it now. Am I right, or do I miss something?

**Comment by Scitator** [comment]
@BloodAxe from my experience, it work perfectly with MacOS, Linux (Ubuntu, Arch), Windows Linux Subsystem. Sorry, but I have no experience with pure windows command line interface - I just managed to use WLS :)

**Review by Scitator — APPROVED** [review]

**Comment by Scitator** [comment]
@BloodAxe Thank you for your PR.
